plugin/src/main/java/com/microsoft/java/bs/gradle/plugin/SourceSetsModelBuilder.java

```diff
@@ -37,6 +39,7 @@
 import com.microsoft.java.bs.gradle.model.impl.DefaultGradleSourceSet;
 import com.microsoft.java.bs.gradle.model.impl.DefaultGradleSourceSets;
 import com.microsoft.java.bs.gradle.plugin.dependency.DependencyCollector;
+import org.jetbrains.annotations.NotNull;
```

Comment:
`org.jetbrains.annotations` will be unavailable in other IDEs.

Reply:
Addressed in https://github.com/microsoft/build-server-for-gradle/pull/173/commits/9120f6841af25c6b698e754f67c36a136ca6cb35